plugins/demoproxytest.px

```diff
@@ -0,0 +1,15 @@
+#!!PLUXY_MAGIC!!
```

Comment:
doesn't match the one in the docs, nor the one actually checked by the demoproxy